### DISCUSSION TITLE
Stephenson 2 DFK 1

### DISCUSSION THREAD

**Discussion by ilkant** [Stellarium/stellarium#3291] [General]
I saw a video and there was said: Stephenson 2-18 is the biggest (known) star in universe. Stellarium search (F3) doesn't recognize that. Please add that star to database.

https://en.wikipedia.org/wiki/Stephenson_2_DFK_1

https://www.facebook.com/reel/975119436858104 (the video)

**Comment by gzotti** [comment]
We don't have such statistical lists. This is what WP or FB are for. The object search via SIMBAD works, however.

**Reply by Crimsoth** [reply]
Add it.

**Reply by alex-w** [reply]
> Add it.

Please search `Gaia DR3 4253084565963481856` in weekly snapshots
